### PR TITLE
docs:  warn users about driver stored in executors

### PR DESCRIPTION
**Pull request by davidbp**

### CODE REVIEW

File: jina/executors/__init__.py [hunk 1]
```diff
@@ -391,8 +391,8 @@ def save(self, filename: str = None):
         .. warning::
             Saved executors will serialize information from its associated `.yaml` file.
             In particular, the drivers are part of what is serialized.  Therefore, if you
-            want to store/load Executors it is encouraged not to change the `.yaml` file after
-            loading, to avoid unexpected behaviours.
+            want to store/load Executors be aware not to change the `.yaml` file after
```
Comment (JoanFM) [inline jina/executors/__init__.py:394]: I would specially talk about drivers.

File: jina/executors/__init__.py [hunk 2]
```diff
@@ -388,6 +388,12 @@ def save(self, filename: str = None):
             All class members created here will NOT be serialized when calling :func:`save`. Therefore if you
             want to store them, please implement the :func:`__getstate__`.
 
+        .. warning::
+            Saved executors will serialize information from its associated `.yaml` file.
+            In particular, information regarthing `drivers` is serialized.
```
Comment (theUnkownName) [inline jina/executors/__init__.py:393]: Typo in "regarding"

File: jina/executors/__init__.py [hunk 3]
```diff
@@ -388,6 +388,12 @@ def save(self, filename: str = None):
             All class members created here will NOT be serialized when calling :func:`save`. Therefore if you
             want to store them, please implement the :func:`__getstate__`.
 
+        .. warning::
+            Saved executors will serialize information from its associated `.yaml` file.
+            In particular, information regarthing `drivers` is serialized.  
+            Therefore, if you want to store/load Executors be aware not to change parts of the  `.yaml` 
+            file that set up `drivers` after loading, any change these parts will not take effect after serialization.
```
Comment (theUnkownName) [inline jina/executors/__init__.py:395]: "any change *in* these parts" maybe?

**Review by theUnkownName — CHANGES_REQUESTED** [review]

File: jina/executors/__init__.py (hunk 3 above)
Comment (nan-wang) [inline jina/executors/__init__.py:395]:
```suggestion
            When you load a stored Executor, be aware not to change the driver settings in the  `.yaml` 
            file. Any change to the driver settings will not take effect after storing Executors.
```

**Review by nan-wang — CHANGES_REQUESTED** [review]
Rephrase it a little bit.

**Pull request closed without merging**